modules/opensearch/opensearch.client.inc

```diff
@@ -554,13 +565,18 @@ function _opensearch_predict_cache(array $collections, TingClientRequest $reques
         $object_request->setProfile($profile);
       }
 
+      // Build both requests for local id and PID.
       $object_id_request = clone $object_request;
       $object_id_request->setObjectId($object->id);
       $local_id_request = clone $object_request;
       $local_id_request->setLocalId($object->localId);
```

Comment:
@cableman I know this is not from this PR, but isn't this a bit dangerous? The localId is not guaranteed to unique over all owners, so we are really not sure this object is the correct cache response for this localID